front/src/account/fields/AccountFieldCompanyDasriDirectTakeOver.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { gql } from "@apollo/client";
+import AccountBooleanField from "./AccountBooleanField";
+
+import {
+  CompanyPrivate,
+  UserRole,
+  MutationUpdateCompanyArgs,
+} from "generated/graphql/types";
+
+import AccountFormCheckboxInput from "./forms/AccountFormCheckboxInput";
+
+type Props = {
+  company: CompanyPrivate;
+};
+
+AccountFieldCompanyDasriDirectTakeOver.fragments = {
+  company: gql`
+    fragment AccountFielCompanySecurityCodeFragment on CompanyPrivate {
```

Comment:
Je ne sais pas si c'est important mais tu as corrigé la typo ailleurs :

```suggestion
    fragment AccountFieldCompanySecurityCodeFragment on CompanyPrivate {
```

Reply:
merci